Code/Blueprint.cs

```diff
@@ -8,18 +8,16 @@ public partial class Blueprint : Button
     [Export]
     private String _title;
 
-    [Export]
-    private Array<Materials> _materials;
-
-    [Export]
-    private Array<int> _quantities;
+    public Dictionary<Materials, int> _recipe;
 
     private List _rootList;
```

Comment:
oooh, I even said it in the PR description, this should be cumulative

Reply:
https://github.com/adrian-azan/Valheim-Todo/pull/3/commits/9fd76698a25a896a32d713b2ea724d4ada5837b6